libs/api/domains/mailchimp/src/lib/dto/mailchimpSubscribe.input.ts

```diff
@@ -0,0 +1,23 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsBoolean, IsOptional, IsString } from 'class-validator'
+
+@InputType()
+export class MailchimpSubscribeInput {
+  @Field()
+  @IsString()
+  signupID!: string
+
+  @Field()
+  @IsString()
+  email!: string
```

Comment:
@IsEmail() to validate email addresses :)

Reply:
Didn't know about that one, thanks :)